pysus/online_data/SIH.py

```diff
@@ -33,7 +33,7 @@ def download(state: str, year: int, month: int, cache: bool = True) -> object:
         fname = f"RD{state}{year2}{month}.dbc"
     if year >= 2008:
         ftype = "DBC"
-        path = f"/dissemin/publicos/SIHSUS/{year}01_/Dados"
+        path = f"/dissemin/publicos/SIHSUS/200801_/Dados"
```

Comment:
This change will break the code for  2009, Moreover the directory `/dissemin/publicos/SIHSUS/200801_/Dados` only contains data from 2011 on.
Year 2010 is only available in xml only